src/docs/product/data-management-settings/server-side-sampling/getting-started.mdx

```diff
@@ -0,0 +1,95 @@
+
+</Note>
+
+Next, go to **[Project] > Settings > Server-side Sampling** in [sentry.io](https://sentry.io) and set a uniform sampling rate for all transactions: this is the number of transactions that Sentry should accept from your application:
+
+![Setting a uniform sampling rate for your project](set-uniform-sampling-rate.png)
+
+Based on your quota and usage, in this step, we’ll suggest sampling rates for both Sentry servers and your Sentry SDK. However, you can enter a custom sampling rate by overriding our recommended values. For example, you can select your current sampling rates to understand how our proposed sampling configuration would impact your transaction volume within the scope of this project. Learn more about [How Sentry Samples](/product/data-management-settings/server-side-sampling/#how-sentry-samples).
+
+The numbers you enter in the “Client” field do not affect your actual SDK sample rate, but if you click “Next” and save, the number you’ve entered in the “Server” field on the “Suggested” row is saved as your uniform server-side sampling rate.
+
+Even though you’ll only set this for one project/SDK, **this rate can affect all the transactions distributed across multiple services**, including other projects set up in Sentry. That’s because server-side currently samples by the context of all other transactions belonging to the same trace. Check out our [full Tracing documentation](/product/sentry-basics/tracing/) for a more detailed explanation.
+
+If this chart doesn’t provide enough information for you to determine a uniform sampling rate for this project, we recommend checking your **Stats** page. The **Stats** page provides a complete overview of how many transactions each project accepts, filters, and drops rather than for just one project.
+
+Once you've entered a uniform sample rate for all transactions, now you need to activate it by clicking “Done” and then toggling the rule to on:
+
+![A sampling rule with the Active toggle highlighted](sampling-rule-toggle.png)
+
+## 3. Set a Sampling Rate Based on a Condition
+
+Now, click “Add Rule” to set more sampling rules based on specific conditions. These rules will help you target the transactions you want to keep and allow you to have more of those transactions sent by the SDK.
+
+When a rule uses multiple conditions, we use the logical expression `AND` to determine when that rule should be evaluated. Multiple values for a condition use a logical or condition for matching. Learn more about [how to create effective server-side sampling rules](/product/data-management-settings/server-side-sampling/#server-side-sampling-configurations).
+
+You can use the following conditions to sample transactions by:
+
+- Environment
+- Release\*
+- User Segment
+
+* Allows glob pattern matching
+
+![Editing a sampling rule based on a release condition](edit-rule-modal.png)
```

Comment:
the screenshots has a black border (from the drop shadow) when it should be transparent there afaict

Reply:
> the screenshots has a black border (from the drop shadow) when it should be transparent there afaict

yeah, these are all just placeholders for now and will all be replaced before this is merged.